packages/dataviews/src/dataviews.js

```diff
@@ -42,11 +48,32 @@ export default function DataViews( {
 		( v ) => v.type === view.type
 	).component;
 	const _fields = useMemo( () => {
-		return fields.map( ( field ) => ( {
-			...field,
-			render: field.render || field.getValue,
-		} ) );
+		return fields.map( ( field ) => {
+			// Normalize formats.
+			field.formats = field.formats || [];
```

Comment:
This looks like a mutation during render, probably something that can cause bugs.

Reply:
you should do something like `const result = { ...field }` and mutate result instead.